src/betterproto/plugin/models.py

```diff
@@ -638,7 +638,10 @@ def proto_name(self) -> str:
 
     @property
     def route(self) -> str:
-        return f"/{self.output_file.package}.{self.parent.proto_name}/{self.proto_name}"
+        package_part = (
+            f"{self.output_file.package}." if self.output_file.package else ""
```

Comment:
I would personally use

```python
package_part = '.'.join([self.output_file.package, self.parent.proto_name])
return f"/{package_part}/{self.proto_name}"
```

But maybe the intention of yours is more clear, since my example almost uses a side effect to achieve the result